README.md

```diff
@@ -6,7 +6,7 @@ There are various different repos for ML research and development but this repo
 The main entry point is
 
 ```
-poetry run python pv_site_production/app.py
+poetry run python pv_site_production/scripts/apply_model.py
```

Comment:
should the main entry point be in scripts? I would have prefered it in `app.py` and then it can be tested. 

In different repos, Ive had scripts directory in the root folder, how come its with in `pv_site_production`

Reply:
This is how I usually organize things, in case there are other entry points later, but I'm fine with reverting back to `app.py` at the root.

Just to be clear, which one do you prefer:
1. `/app.py`
2. `/pv_ste_production/app.py`
3. `/scripts/app.py`
?

Number 2. right?

Reply:
number 2. please. But if there is a different way that is more common, then just say

Reply:
I'm not sure what is more common. 2 it is, at least for now!